docs/resources/User.md

```diff
@@ -28,7 +28,8 @@ There are other rules and restrictions not shared here for the sake of spam and
 | ------------- | --------- | ---------------------------------------------------------------------------------------------------- | --------------------- |
 | id            | snowflake | the user's id                                                                                        | identify              |
 | username      | string    | the user's username, not unique across the platform                                                  | identify              |
-| discriminator | string    | the user's 4-digit discord-tag                                                                       | identify              |
+| discriminator | string    | the user's Discord-tag                                                                               | identify              |
+| global_name   | ?string   | the user's display name, if it is set. For bots, this is the application name                        | identify              |
```

Comment:
Is the property nullable just during the migration or an user may not have a global name even after migration?

Reply:
> As part of the new username system, standard Discord users can define a non-unique display name. This value will be a new `global_name` field with a max length of 32 characters. If the user has not set a display name, `global_name` will be null.

https://github.com/discord/discord-api-docs/blob/b7a4d80861aea9e116d123a253acd460386a89b5/docs/Change_Log.md#L41